js/web/script/test-runner-cli-args.ts

```diff
@@ -152,7 +152,7 @@ export interface TestRunnerCliArgs {
   wasmFlags?: Env.WebAssemblyFlags;
   webglOptions?: InferenceSession.WebGLExecutionProviderOption;
   webglFlags?: Env.WebGLFlags;
-
+  globalFlags?: Env;
```

Comment:
I think it's a good time to merge `webglFlags` and `wasmFlags` into `globalFlags` (maybe rename it to just 'flags'?).

the reason why previously we need separated flags is because we use code-gen to set those flags but we are going to change this part (#7688).

Reply:
rebased and merged